data/domain/world-6/summoning.tsx

```diff
@@ -144,25 +193,47 @@ export class BattlesInfo {
 
     constructor() {}
 
+    // This should now be used anywhere this value is needed
+    getTotalVictories = (): number => {
+        return this.allVictories.reduce((sum, victories) => sum + victories, 0);
+    }
+
     static getBattleBonusText = (battle: SummonEnemyBonusModel | undefined, bonusValue: number): string => {
         if (battle) {
-            const bonus = 3.5 * bonusValue;
-            switch (battle.bonusId) {
-                // Only those bonuses have a '{' to be replaced by the bonus value, others use a '<'
-                case 5:
-                case 10:
-                case 12:
-                case 13:
-                case 19:
-                case 20:
-                    return battle.bonus.replace(/{/, nFormatter(bonus));
+            var bonus = 0;
+            switch (true)
+            {
+                // +1 from in-game indexes as it starts from 1
+                case battle.bonusId == 21:
+                case battle.bonusId == 23:
+                case battle.bonusId == 25:
+                case battle.bonusId == 32:
+                    bonus = bonusValue;
+                case battle.bonusId == 20:
+                    bonus = 3.5 * bonusValue;
```

Comment:
do you even need a case for it? it's the same as the default 🤔

Reply:
Yeah, I just kept it is in the game code, but can surely clean it up a bit

Reply:
On second look, I kept it as is, because in the "real" getBonus function those separate case have a real reason to be there, so I wanted to keep the same case to mirror it

But if you still prefer it to be "cleaned" it can easily be done

Reply:
Nah it's all good, sometimes keeping things in sync makes total sense.